terragrunt/aws/ecr/iam_oidc.tf

```diff
@@ -98,4 +98,25 @@ data "aws_iam_policy_document" "ecr_image_manage" {
     ]
     resources = ["*"]
   }
+}
+
+data "aws_iam_policy_document" "pr_review_env_policy_document" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
+
+    resources = [
+      "arn:aws:lambda:${var.region}:${var.account_id}:function:pr-review-env-*"
+    ]
+  }
+}
+
+resource "aws_iam_role" "pr-review-env-lambda" {
+  name               = "pr-review-env"
+  assume_role_policy = data.aws_iam_policy_document.pr_review_env_policy_document.json
```

Comment:
For this, you'll ned to create a new `aws_iam_policy_document` that defines the assume role policy.  Here's an example of what that looks like:
https://github.com/cds-snc/list-manager/blob/b7199781f8736cc821d8a8f0d6addaa91d5fa123/terragrunt/aws/api/iam.tf#L1-L12